docs/integrations/github/discovery.md

```diff
@@ -19,13 +19,17 @@ to the catalog configuration:
 ```yaml
 catalog:
   locations:
+    # (since 0.13.5) Scan all repositories for a catalog-info.yaml in the root of the default branch
     - type: github-discovery
-      target: https://github.com/myorg/service-*/blob/main/catalog-info.yaml
+      target: https://github.com/myorg
+    # Or use a custom pattern for discovering catalog entity descriptor files
+    - type: github-discovery
+      target: https://github.com/myorg/service-*/blob/main/your-own-format.yaml
```

Comment:
Maybe you could explicitly add an example of using `-` here too. As we'd prefer to use that entirely over selecting a branch if possible to reduce errors caused by the same entity from two different locations :)

Reply:
Good idea. I added some more examples. Intuïtively, you should be able to deduce the options by looking at the examples, I think.